openerp/http.py

```diff
@@ -542,7 +543,39 @@ def dispatch(self):
         if self.jsonp_handler:
             return self.jsonp_handler()
         try:
+            rpc_request = logging.getLogger(__name__ + '.rpc.request')
+            rpc_response = logging.getLogger(__name__ + '.rpc.response')
+            rpc_request_flag = rpc_request.isEnabledFor(logging.DEBUG)
+            rpc_response_flag = rpc_response.isEnabledFor(logging.DEBUG)
+
+            endpoint = self.endpoint.method.__name__
```

Comment:
these variables (endpoint, model, method, args) are unused when *debug_rpc* is disabled, couldn't they be pushed inside the next `if`?